### PR TITLE
Update nav with new branding

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Update nav with new branding

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8037
- Check that the global nav works as expected

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5635

### CODE REVIEW

File: static/js/global-nav.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
-import { createNav } from "@canonical/global-nav";
+// import { createNav } from "@canonical/global-nav";
 
-createNav({ maxWidth: "72rem" });
+// createNav({ maxWidth: "72rem" });
```
Comment (carkod) [inline static/js/global-nav.js:3]: Shouldn't we remove this file entirely if we have everything commented out?

**Comment by carkod** [comment]
Probably not related to this PR, but when you scroll down with the global-nav opened, there is a gap that doesn't have the shade:
gap-shade

However if you compare it with current global-nav, it doesn't have it.

global-nav-shade

**Comment by davegoddard42** [comment]
PdM has seen this and is happy.

**Comment by spencerbygraves** [comment]
@mtruj013 Lyubo did a quick comparison to the SVGs for me and the orange tag is a bit too wide. Looks like we would need to fix this in Vanilla. (He changed the text to compare like for like).

image

**Comment by spencerbygraves** [comment]
I've opened an [issue in Vanilla](https://github.com/canonical/vanilla-framework/issues/4585) to look at the spacing mentioned above.

**Pull request merged**